### PR TITLE
Ds fixing bugs01

### PR DESCRIPTION
**Pull request by DStrelak**

### CODE REVIEW

**Comment by dmaluenda** [comment]
I have not much idea about this. @cossorzano can you have a look on this? Thx!

File: java/src/xmipp/viewer/scipion/ScipionMetaData.java [hunk 1]
```diff
@@ -171,30 +164,22 @@ public void loadData() {
...
-            rs.close();
-            stmt.close();
-            c.close();
             StopWatch.getInstance().printElapsedTime("loaded data");
         } catch (Exception e) {
             throw new IllegalArgumentException(e);
-            
-
+        } finally {
+            try {if (null != rs) rs.close(); } catch (SQLException e) {/* suppress */}
```
Comment (pconesa) [inline java/src/xmipp/viewer/scipion/ScipionMetaData.java]: Shouldn't we close the stmt and the connection too? in the finally

**Comment by pconesa** [comment]
what was the bug?

File: java/src/xmipp/viewer/scipion/ScipionMetaData.java [hunk 2]
```diff
@@ -1229,39 +1198,30 @@ public static boolean isScipionMetaData(String filename)
...
-	    		exists = rs.next();
-	    		if (!exists)
-	    			result = false;
+	            try (ResultSet rs1 = stmt.executeQuery(query)) {
+    	    		exists = rs1.next();
+    	    		if (!exists)
+    	    			result = false;
+    			}
     		}
-            rs.close();
```
Comment (pconesa) [inline java/src/xmipp/viewer/scipion/ScipionMetaData.java]: Shouldn't we close thing here?

File: java/src/xmipp/viewer/scipion/ScipionMetaData.java [hunk 3]
```diff
@@ -569,12 +549,10 @@ public void writeBlock(String path) {
...
             stmt.executeUpdate(sql);
             c.commit();
-            stmt.close();
-            c.close();
-
         } catch (Exception e) {
             e.printStackTrace();
-
+        } finally {
+            try { if (null != stmt) stmt.close(); } catch (SQLException e) {/* suppress */}
```
Comment (pconesa) [inline java/src/xmipp/viewer/scipion/ScipionMetaData.java]: Connection close?

**Comment by pconesa** [comment]
In agreement with @DStrelak , we are going to close this and reopen it once the release is done.

**Pull request closed without merging**